xrdsst/controllers/service.py

```diff
@@ -327,6 +374,32 @@ def remote_update_service_parameter(ss_api_config, service_description_conf, ser
         except ApiException as err:
             BaseController.log_api_error('ServicesApi->update_service', err)
 
+    def remote_list_service_descriptions(self, ss_api_config, security_server, client):
+        clients_api = ClientsApi(ApiClient(ss_api_config))
+        try:
+            service_descriptions_list = []
+            render_data = []
+            client_ids = client.split(',')
+            for client_id in client_ids:
+                service_descriptions = clients_api.get_client_service_descriptions(id=client_id)
+                for service_description in service_descriptions:
+                    service_descriptions_list.append({'security_server': security_server["name"],
+                                                      'client_id': client_id,
+                                                      'description_id': service_description.id,
+                                                      'url': service_description.url,
+                                                      'type': service_description.type,
+                                                      'disabled': service_description.disabled,
+                                                      'services': len(service_description.services)})
```

Comment:
Maybe here check if the services exists just in case, something like: len(service_description.services) if service_description.services else 0

Reply:
Added the check